test/e2e/specs/editor/blocks/navigation.spec.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe(
+	'As a user I want the navigation block to fallback to the best possible default',
+	() => {
+		test.beforeEach( async ( { admin } ) => {
+			await admin.createNewPost();
+		} );
+
+		test( 'default to a list of pages if there are no menus', async ( {
+			editor,
+		} ) => {
+			await editor.insertBlock( { name: 'core/navigation' } );
+
+			// Check Page List is in the list view.
+
+			// Open the list view.
+			await editor.page.locator( '[aria-label="List View"i]' ).click();
+			// Click the Navigation block expander, we need to use force because it has aria-hidden set to true.
+			await editor.page
+				.locator(
+					`//a[.//span[text()='Navigation']]/span[contains(@class, 'block-editor-list-view__expander')]`
+				)
+				.click( { force: true } );
+			// Find the Page list selector inside the navigation block.
+			const pageListSelector = await editor.page.locator(
+				`//table[contains(@aria-label,'Block navigation structure')]//a[.//span[text()='Page List']]`
+			);
+
+			expect( pageListSelector ).toBeTruthy();
```

Comment:
I personally don't think there's any need to check List View here, checking the editor content is enough, and there could be (or are) separate tests for List View checking that its content matches the block editor content. It'll also help the tests be less flaky if there are fewer steps.